### PR TITLE
feat: swap analytics for LLM

### PR DESCRIPTION
**Pull request by cjordan-ledger**
### 📝 Description

LLM Swap Analytics

### ❓ Context

- **Impacted projects**: Mobile
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-4833

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/FabActions/index.tsx [hunk 1]
```diff
@@ -84,6 +84,7 @@ export const FabButtonBarProvider = ({
   eventProperties?: { [key: string]: unknown };
   children: (value: { quickActions: ActionButtonProps[] }) => ReactNode;
 }) => {
+  const { track } = useAnalytics();
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/components/FabActions/index.tsx:87]: Whiat is different between?
```
import { track } from "../../analytics";
``` 
and 
```
const { track } = useAnalytics()
```

File: apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx [hunk 2]
```diff
@@ -123,9 +124,14 @@ export default function useAssetActions({
             ...(availableOnSwap
               ? [
                   {
-                    id: "swap",
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx:126]: Why are we removing this it? Can we use the same structure for any item in the array?

File: apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx [hunk 3]
```diff
@@ -73,6 +76,12 @@ export default function SelectFeesStrategy({
 
   const onPressStrategySelect = useCallback(
     (item: FeeStrategy) => {
+      track("button_clicked", {
+        button: item.label,
+        flow: "swap",
+        page: "Swap quotes",
+        swapVersion: SWAP_VERSION,
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx:83]: Can we create a default swap property object an export it across any track? (If tomorrow, we want to change the property the change will be done only in one place)
swapVersion, flow

File: apps/ledger-live-mobile/src/screens/Swap/Form/Max.tsx [hunk 4]
```diff
@@ -2,9 +2,20 @@ import React from "react";
...
 
 export function Max({ swapTx }: { swapTx: SwapTransactionType }) {
   const { t } = useTranslation();
+  const { track } = useAnalytics();
+
+  const onToggle = (event: boolean) => {
+    track("button_clicked", {
+      button: "max",
+      flow: "swap",
+      state: event,
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/screens/Swap/Form/Max.tsx:15]: swap version?

File: apps/ledger-live-mobile/src/components/FabActions/index.tsx (hunk 1 above)
Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/components/FabActions/index.tsx:87]: Good question. I used `useAnalytics` as it calls `track` and adds in the current `page` route automatically. However, when I've been testing this, `page` is sometimes wrongly assigned to `portfolio_navigator`.  I also have just noticed that `track`  also adds in a `screen` value which does look correct.

 It's a little bit confusing. I think we should send either `page` or `screen` and ensure that this is set to the right value, but don't really want to fix this issue in this PR as it'll affect all analytics.

File: apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx (hunk 3 above)
Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx:83]: Can do, I thought about this but wasn't sure if it was worthwhile as there will only be two fields in that shared object.

File: apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx (hunk 2 above)
Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx:126]: `apps/ledger-live-mobile/src/components/FabActions/index.tsx:127`

We want to add additional event properties like `swapVersion` etc, so we need to replace `id` with `event` in order to do that.

File: apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx (hunk 3 above)
Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx]: https://github.com/LedgerHQ/ledger-live/pull/2255/commits/6bf6cb751e5882133d7cc8a09edf24853bbf8fc9

**Review by sarneijim — APPROVED** [review]

**Pull request merged**